scripts/update_config.js

```diff
@@ -51,6 +51,9 @@ console.info('Starting configuration migration...');
 const oldConfig = JSON.parse(fs.readFileSync(oldConfigPath, 'utf8'));
 const newConfig = JSON.parse(fs.readFileSync(newConfigPath, 'utf8'));
 
+newConfig.api.ssl = extend(true, {}, oldConfig.ssl);
```

Comment:
newConfig.api.ssl = extend(true, newConfig.api.ssl, oldConfig.ssl); ?

Reply:
There is no change in the structure of the `ssl` objet itself in previous and current release. So this code will also work fine. 
```
newConfig.api.ssl = extend(true, {}, oldConfig.ssl);
```